src/types/FullTodo.ts

```diff
@@ -0,0 +1,6 @@
+import { Todo } from './Todo';
+import { User } from './User';
+
+export interface FullTodo extends Todo {
```

Comment:
Interface `FullTodo` extends `Todo` but it is not clear what additional properties it adds. Consider renaming the interface to better reflect its purpose. 🤔